Sileo/Backend/Repo Manager/RepoManager.swift

```diff
@@ -10,7 +10,8 @@ import Foundation
 
 // swiftlint:disable:next type_body_length
 final class RepoManager {
-    
+    // This check is here because it's used in multiple places throughout the code
+    private final let isProcursus = FileManager.default.fileExists(atPath: "/etc/apt/trusted.gpg.d/memo.gpg")
```

Comment:
This is a bad check. Use /.procursus_strapped
Also doesn’t account for macCatalyst, which doesn’t have that *and* doesn’t have /etc/blah/blah

Reply:
There were no conflicting issues with macCatalyst that had neither which is why I didn't need to check but yes I'll change my check.

Reply:
I don’t know what “conflicting issues with macCatalyst” you mean. I’m just saying, if this is a Procursus check, then it should apply to macOS as well. Otherwise, it’s misleading. isMobileProcursus?

Reply:
Called it isMobileProcursus, sorry for the confusion

Reply:
It’s fine lol, that’s what code review is for.